src/v2/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx

```diff
@@ -57,3 +81,11 @@ export const ThankYou: React.FC = () => {
     </>
   )
 }
+
+export const ThankYouFragmentContainer = createFragmentContainer(ThankYou, {
+  submission: graphql`
+    fragment ThankYou_submission on ConsignmentSubmission {
+      userEmail
+    }
+  `,
+})
```

Comment:
Are you sure that it works correctly? This query should fail since the user shouldn't be able to load submissions with the `submitted` state. I suppose it may work if the information comes from the Relay store, but if information comes from the server we should fix that.

Reply:
it looks like it works, I've added a video in the description.

Reply:
very interesting, I'll check it locally tomorrow.

Reply:
I suppose it happens because you have admin rights.

Reply:
thank you. do we not allow querying 'submitted' submissions, by the way? I wonder where this restriction is set.

Reply:
Yes, we allow loading or updating the submission only if the submission has draft status. the logic here: https://github.com/artsy/convection/blob/4fd180fc614b67089eb3f203ffff329525481793/app/graphql/resolvers/submission_resolver.rb#L9

Reply:
thank you for this, I had no idea. I'll test it with a non-admin user tomorrow morning.

Reply:
tried with non-admin user: correct, no submission is received.